BICCN-policies/dataset-metadata.yaml

```diff
@@ -0,0 +1,85 @@
+identifier: REQUIRED ## Post upload (or during dandi organize)
+name: REQUIRED
+description: REQUIRED
+contributors: # required for author and contact
+- orcid: # REQUIRED
+  roles: # REQUIRED from https://casrai.org/credit/ + maintainer, contact
+  email: Recommended # filled from orcid
+  name: Recommended  # filled from orcid
+  affiliations: optional  # filled from orcid
+sponsors:
+- identifier: RECOMMENDED
+  name: REQUIRED
+  url: RECOMMENDED
+license:
+- url # REQUIRED
+keywords: [key1, key2,]
```

Comment:
Related to keywords - we would want to capture the modality and relevant technique(s) for BICCN purposes.